os/board/rtl8730e/src/component/mbed/targets/hal/rtl8730e/i2s_api.c

```diff
@@ -260,8 +210,8 @@ void i2s_init(i2s_t *obj, PinName sck, PinName ws, PinName sd_tx, PinName sd_rx,
 void i2s_set_dma_buffer(i2s_t *obj, char *tx_buf, char *rx_buf,
 						uint32_t page_num, uint32_t page_size)
 {
-	//uint8_t i2s_idx = obj->i2s_idx;
-	u32 i, j;
+	uint8_t i2s_idx = obj->i2s_idx;
```

Comment:
`i2c_idx` is used as index of array. I think `obj` and `i2s_idx` should be checked ahead to avoid invalid memory access because it is assigned by parameter.

Reply:
I will place another check here.